dgraph/cmd/root.go

```diff
@@ -31,7 +31,6 @@ import (
 	"github.com/dgraph-io/dgraph/dgraph/cmd/zero"
 	"github.com/dgraph-io/dgraph/x"
 	"github.com/spf13/cobra"
-	flag "github.com/spf13/pflag"
```

Comment:
Aliasing `pflag` => `flag` and `flag` => `goflag` created unnecessary ambiguity. I've changed this to remove the ambiguity.